loqusdbapi/models.py

```diff
@@ -65,6 +76,7 @@ class Variant(BaseVariant):
 
 
 class StructuralVariant(BaseVariant):
+    id: Optional[PydanticObjectId] = Field(alias="_id")
```

Comment:
Why doesn't it work if you have BsonObjectId in there, like this?
`id: Optional[BsonObjectId] = Field(alias="_id")`

Reply:
Since that is not a Pydantic type I guess?

Reply:
Ah OK

Reply:
I think it has to be one of the actual Pydantic field_types there. Why Any doesn't work is beyond me at the moment; something with how the ObjectId is made.

Reply:
Finally... Thanks for the coaching.

Reply:
Perhaps it means Any of the pydantic types